tests/aurora/nfofile.cpp

```diff
@@ -0,0 +1,112 @@
+		0x09, 0x00, 0x00, 0x00, 0x3c, 0x00, 0x00, 0x00, 0x0a, 0x00, 0x00, 0x00,
+		0x0a, 0x00, 0x00, 0x00, 0x40, 0x00, 0x00, 0x00, 0x0a, 0x00, 0x00, 0x00,
+		0x0b, 0x00, 0x00, 0x00, 0x44, 0x00, 0x00, 0x00, 0x0a, 0x00, 0x00, 0x00,
+		0x0c, 0x00, 0x00, 0x00, 0x48, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+		0x0d, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x0b, 0x00, 0x00, 0x00,
+		0x0e, 0x00, 0x00, 0x00, 0x4c, 0x00, 0x00, 0x00, 0x41, 0x52, 0x45, 0x41,
+		0x4e, 0x41, 0x4d, 0x45, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+		0x4c, 0x41, 0x53, 0x54, 0x4d, 0x4f, 0x44, 0x55, 0x4c, 0x45, 0x00, 0x00,
+		0x00, 0x00, 0x00, 0x00, 0x54, 0x49, 0x4d, 0x45, 0x50, 0x4c, 0x41, 0x59,
+		0x45, 0x44, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x43, 0x48, 0x45, 0x41,
+		0x54, 0x55, 0x53, 0x45, 0x44, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+		0x53, 0x41, 0x56, 0x45, 0x47, 0x41, 0x4d, 0x45, 0x4e, 0x41, 0x4d, 0x45,
+		0x00, 0x00, 0x00, 0x00, 0x47, 0x41, 0x4d, 0x45, 0x50, 0x4c, 0x41, 0x59,
+		0x48, 0x49, 0x4e, 0x54, 0x00, 0x00, 0x00, 0x00, 0x53, 0x54, 0x4f, 0x52,
+		0x59, 0x48, 0x49, 0x4e, 0x54, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+		0x4c, 0x49, 0x56, 0x45, 0x31, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+		0x00, 0x00, 0x00, 0x00, 0x4c, 0x49, 0x56, 0x45, 0x32, 0x00, 0x00, 0x00,
+		0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x4c, 0x49, 0x56, 0x45,
+		0x33, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+		0x4c, 0x49, 0x56, 0x45, 0x34, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+		0x00, 0x00, 0x00, 0x00, 0x4c, 0x49, 0x56, 0x45, 0x35, 0x00, 0x00, 0x00,
+		0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x4c, 0x49, 0x56, 0x45,
+		0x36, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+		0x4c, 0x49, 0x56, 0x45, 0x43, 0x4f, 0x4e, 0x54, 0x45, 0x4e, 0x54, 0x00,
+		0x00, 0x00, 0x00, 0x00, 0x50, 0x4f, 0x52, 0x54, 0x52, 0x41, 0x49, 0x54,
+		0x30, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x1b, 0x00, 0x00, 0x00,
+		0x45, 0x6e, 0x64, 0x61, 0x72, 0x20, 0x53, 0x70, 0x69, 0x72, 0x65, 0x20,
+		0x2d, 0x20, 0x4b, 0x6f, 0x6d, 0x6d, 0x61, 0x6e, 0x64, 0x6f, 0x6d, 0x6f,
+		0x64, 0x75, 0x6c, 0x09, 0x00, 0x00, 0x00, 0x45, 0x4e, 0x44, 0x5f, 0x4d,
+		0x30, 0x31, 0x41, 0x41, 0x04, 0x00, 0x00, 0x00, 0x74, 0x65, 0x73, 0x74,
+		0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+		0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+		0x08, 0x70, 0x6f, 0x5f, 0x70, 0x6d, 0x68, 0x61, 0x34, 0x00, 0x00, 0x00,
+		0x00, 0x01, 0x00, 0x00, 0x00, 0x02, 0x00, 0x00, 0x00, 0x03, 0x00, 0x00,
+		0x00, 0x04, 0x00, 0x00, 0x00, 0x05, 0x00, 0x00, 0x00, 0x06, 0x00, 0x00,
+		0x00, 0x07, 0x00, 0x00, 0x00, 0x08, 0x00, 0x00, 0x00, 0x09, 0x00, 0x00,
+		0x00, 0x0a, 0x00, 0x00, 0x00, 0x0b, 0x00, 0x00, 0x00, 0x0c, 0x00, 0x00,
+		0x00, 0x0d, 0x00, 0x00, 0x00, 0x0e, 0x00, 0x00, 0x00
+};
+
+GTEST_TEST(NFOFile, getPortrait) {
+	const Aurora::NFOFile nfo(new Common::MemoryReadStream(kNFOFile));
+
+	ASSERT_EQ(nfo.getPortrait0(), "po_pmha4");
```

Comment:
For strings, you should use STREQ instead of EQ, in combination with c_str(). This gives better/clearer warning messages should the test fail (i.e. when it fails, it can print the actual string).

Also, use EXPECT_* instead of ASSERT_* where possible. If an ASSERT_* fails, that whole function will abort, while EXPECT_* will continue with the following lines. So with EXPECT_*, you're going to see if further tests also fail.

ASSERT_* are needed if the lines below depend on the test to succeed, for example when it would return a pointer that is then dereferenced. See also https://github.com/google/googletest/blob/master/googletest/docs/Primer.md#assertions .

This line should then be
`EXPECT_STREQ(nfo.getPortrait0().c_str(), "po_pmha4");`

Likewise for the other lines below.